src/registry.js

```diff
@@ -0,0 +1,296 @@
+"use strict";
```

Comment:
It'd be useful to keep all those changes separate from the core polyfill though, rather than mixing them all in together. If we can. If you've got examples where we can't possibly make the changes separately then that's ok too, but we should document those, so we can work out what's going on here in future.